extensions/amp-ad-network-smartadserver-impl/0.1/amp-ad-network-smartadserver-impl.js

```diff
@@ -0,0 +1,250 @@
+/* eslint-disable local/no-forbidden-terms */
```

Comment:
Can you find out why is this needed? I do not believe you can disable a check for an entire file (it will fail some other tests). It will be better to fix the underlying issue.

Reply:
Removed. It was needed just for debugging.